modal_proto/api.proto

```diff
@@ -623,6 +623,7 @@ message DataChunk {
     string data_blob_id = 3;
   }
   uint64 index = 4; // Index of this data chunk in the stream.
+  optional bool is_eof = 5; // Used for sandbox stdin 
```

Comment:
Added this so that I could reuse the same `DataChannel` used for storing user's `stdin` (`ContainerInput`) in interactive mode for storing Sandbox's `stdin.write` and `stdin.write_eof` data.